R/moduleCoverage.R

```diff
@@ -0,0 +1,114 @@
+###############################################################################
+#' Calculate module coverage of unit tests
+#'
+#'
+#' Calculate the test coverage by unit tests for the module and functions in module
+#'
+#'
+#' @param name  Character string. The module's name.
+#'
+#' @param path  Character string. Subdirectory in which to place the new module code file.
+#'              The default is the current working directory.
+#'
```

Comment:
i'm not sure I understand why this needs to be specified. coverage testing can be done entirely in a temp dir, can't it?